sys/net/gnrc/network_layer/ipv6/nib/_nib-slaac.c

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
```

Comment:
I don't think we have a policy on that.

```sh
git grep --name-only '/\*\* @} \*/' -- *.c | xargs tail -n 1 | grep "/\*\* @} \*/" | wc -l
```

yields 176 files that do it on the last line (but I bet there are some false negatives, that do not show). I also think it isn't that important. Its more important that the syntax checks out (and even that isn't checked, since we basically only use Doxygen in C-files for consistency with the headers, but it isn't parsed).

Reply:
fine